packages/create-cloudflare/src/cli.ts

```diff
@@ -30,6 +32,39 @@ const WRANGLER_DEFAULTS = {
 export const main = async (argv: string[]) => {
 	const args = await parseArgs(argv);
 
+	// Print a newline
+	logRaw("");
+
+	if (args.autoUpdate && (await isUpdateAvailable())) {
+		await runLatest();
+	} else {
+		await runCli(args);
+	}
+};
+
+const isUpdateAvailable = async () => {
+	if (process.env.VITEST) {
+		return false;
+	}
+
+	// Use a spinner when running this check since it may take some time
+	const s = spinner(spinnerFrames.vertical);
```

Comment:
why different style spinner for this one? is there a semantic difference between waiting on this step vs waiting on other steps that you are trying to convey?

Reply:
The current spinner uses the same characters that we use for starting, stopping, and continuing sections (the top and bottom corners, and the gray bars to the left). It feels a little out of place when not inside one of those contexts

Reply:
I see. ok. sounds good.